assets/styles/style.scss

```diff
@@ -47,6 +47,7 @@ This files table contents are outlined below>>>>>
 ** - Coming Soon CSS
 ** - 404 Error CSS
 ** - Footer CSS
+** - Custom CSS
```

Comment:
lol i didn't realise there was this sass file too, i've been changing css in just the `.css` file - we might need to consolidate these css files later